media-libs/libjxl/libjxl-0.7.0_pre20220329.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_ECLASS=cmake
```

Comment:
Can drop this line. `cmake` is the default.

Reply:
I will do it.